specification/electronic-prescription-service-api.yaml

```diff
@@ -246,15 +246,55 @@ paths:
                 value:
                   $ref: components/examples/example-1-repeat-dispensing/SendRequest-FhirMessageSigned.json
         description: ''
+      responses:
+        '202':
+          description: Accepted
+          headers:
+            content-location:
```

Comment:
Again, not sure if this is case sensitive, but we return Content-Location in our response

Reply:
This is how Hapi returns them